demisto_sdk/commands/content_graph/commands/update.py

```diff
@@ -68,58 +99,64 @@ def update_content_graph(
         use_git (bool): Whether to use git to get the packs to update.
         imported_path (Path): The path to the imported graph.
-        use_local_import (bool): Whether to use the current import folder to import graph.
         packs_to_update (List[str]): The packs to update.
         dependencies (bool): Whether to create the dependencies.
         output_path (Path): The path to export the graph zip to.
     """
-    if not use_local_import and not imported_path and not use_git:
+    if not imported_path and not use_git:
         logger.info("No arguments were given, using git")
         use_git = True
+    if output_path:
+        output_path = output_path / marketplace.value
+    content_graph_interface.export_graph(output_path, override_commit=use_git)
+
     git_util = GitUtil()
+    is_external_repo = is_external_repository()
 
-    if is_external_repository():
+    if is_external_repo:
         packs_to_update = get_all_repo_pack_ids()
     packs_to_update = list(packs_to_update) if packs_to_update else []
     builder = ContentGraphBuilder(content_graph_interface)
     if not should_update_graph(
-        content_graph_interface, git_util, imported_path, use_local_import
+        content_graph_interface, use_git, git_util, imported_path, packs_to_update
     ):
         logger.info(
-            f"Content graph is up to date, no need to update. UI representation is available at {NEO4J_DATABASE_HTTP} "
+            f"Content graph is up to date, no need to update. Make sure to add/commit your changes. UI representation is available at {NEO4J_DATABASE_HTTP} "
             f"(username: {NEO4J_USERNAME}, password: {NEO4J_PASSWORD})"
         )
+        content_graph_interface.export_graph(output_path, override_commit=use_git)
+
         return
     builder.init_database()
-    if not use_local_import:
+    if imported_path:
+        # Import from provided path
+        logger.info(f"Importing graph from {imported_path}")
+        # we need to clean the import dir since we import from provided path
         content_graph_interface.clean_import_dir()
-        if not imported_path:
-            # getting the graph from remote, so we need to clean the import dir
-            try:
-                extract_remote_import_files(content_graph_interface)
-            except RuntimeError as e:
+        content_graph_interface.import_graph(imported_path)
+
+    else:
+        # Try to import from local folder
+        logger.info(f"Importing graph from {content_graph_interface.import_path}")
+        # no need to clean the import dir, since we are importing from it
+        is_graph_up_to_date = content_graph_interface.import_graph(None)
+
+        if not is_graph_up_to_date:
+            # Import from remote if local failed
+            logger.info("Importing graph from bucket")
+            import_succeeded = import_from_bucket(
+                content_graph_interface, is_external_repo
+            )
+
+            if not import_succeeded:
```

Comment:
I think all this could be replaced with:
```
success = content_graph_interface.import_graph(imported_path)
if not success and not imported_path:
    success = content_graph_interface.import_graph(download=True, fail_on_error=is_external_repo)
```
this requires moving `extract_remote_import_files()` to the interface/import handler, as well as calling  `clean_import_dir()` from within `import_graph()` (rather than `update_content_graph()` which is a more high-level method and shouldn't call these methods directly), according to its arguments.